### PR TITLE
feat(gcp): Adds GCP tags docs

### PR DESCRIPTION
**Pull request by tfelix**
Sorts docs a bit and smaller improvements in type case.

Concerns: CU-82wm66

### CODE REVIEW

**Comment by meshcloud-gmbh** [comment]
Task linked: [CU-82wm66 Tag replication GCP Projects](https://app.clickup.com/t/82wm66)

File: docs/meshstack.tag-schema.md [hunk 1]
```diff
@@ -25,7 +25,124 @@ such metadata include:
...
 
-## Consuming Metadata in Landing Zones
+## Tag Schemas
+
+[Configured tag-schemas](meshstack.tag-schema.md) describe the tags available on each meshModel entity and
+their associated validation rules. These rules can describe for example if a tag is required or optional.
+
+> meshStack currently supports tag schemas on [meshCustomers](meshcloud.customer.md) and [meshProjects](meshcloud.project.md) only.
+
+For each supported entity, meshStack can maintain two separate separate tag-schemas: **unrestricted** and **restricted** tags.
```
Comment (chewbeccy) [inline docs/meshstack.tag-schema.md:35]:
```suggestion
For each supported entity, meshStack can maintain two separate tag-schemas: **unrestricted** and **restricted** tags.
```

File: docs/meshstack.tag-schema.md [hunk 2]
```diff
@@ -25,7 +25,124 @@ such metadata include:
...
+
+### Unrestricted Tags
+
+Entity owners can edit tag values for **unrestricted tags** in self-service via meshPanel. For example, a user with a [Customer Admin](./meshcloud.customer.md#assign-meshcustomer-roles) role can edit tags values on the [meshCustomer](meshcloud.customer.md) as well as the [meshProjects](meshcloud.project.md) owned by that customer.
+
+### Restricted Tags
+
+Only [Partners](./administration.index.md) can edit tag values for **restricted tags**. This is useful if an organization
```
Comment (chewbeccy) [inline docs/meshstack.tag-schema.md:43]: Good point. But as far as I know partners can not change tag values through the meshPanel in general. Mostly the values can be changes by authoritative external systems.

File: docs/meshstack.tag-schema.md [hunk 3]
```diff
@@ -25,7 +25,124 @@ such metadata include:
...
+
+## Configuring Tag Schemas
+
+### Defining Tag Schemas
+
+Operators define tag-schemas using JSON Schema. As described earlier on this page, meshStack uses metadata tags in a wide range of use
+cases like providing parameters to Landing Zones. To ensure compatibility with different interfaces, tags must be simple
+key-value pairs of strings only.
+
+The tag schema must therefore ensure all values are `string` values only. Operators that wish to represent numbers can
```
Comment (chewbeccy) [inline docs/meshstack.tag-schema.md:55]:
```suggestion
The tag schema must therefore ensure all values are `string` values only. Operators who want to represent numbers can
```

**Review by chewbeccy — COMMENTED** [review]
added some comments

File: docs/meshstack.tag-schema.md (hunk 1 above)
Comment (tfelix) [reply docs/meshstack.tag-schema.md:35]: done

File: docs/meshstack.tag-schema.md (hunk 2 above)
Comment (tfelix) [reply docs/meshstack.tag-schema.md:43]: I just copy pasted the stuff :D I am not sure but I change the text accordingly.

File: docs/meshstack.tag-schema.md (hunk 3 above)
Comment (tfelix) [reply docs/meshstack.tag-schema.md:55]: done

**Review by chewbeccy — APPROVED** [review]

**Pull request merged**